roles/accprovision/templates/net-operator-cr.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: aci.ctrl/v1alpha1
+kind: AccProvisionInput
+metadata:
+  name: accprovisioninput
+  namespace: aci-containers-system
+spec:
+  acc_provision_input:
+    aci_config:
+      system_id: mykube
+      apic_hosts:
+      - 10.1.1.101
+      vmm_domain:
+        encap_type: vxlan
+        mcast_range:
+          start: 225.20.1.1
+          end: 225.20.255.255
```

Comment:
L12-16 should be removed

Reply:
removing assuming backend acc-provision supports the removal